packages/lab/src/overlay/OverlayPanel.tsx

```diff
@@ -17,6 +16,7 @@ export interface OverlayPanelProps extends ComponentPropsWithoutRef<"div"> {}
 export const OverlayPanel = forwardRef<HTMLDivElement, OverlayPanelProps>(
   function OverlayPanel(props, ref: ForwardedRef<HTMLDivElement>) {
     const { className, ...rest } = props;
+    const ariaLabelledBy: string | undefined = props["aria-labelledby"];
```

Comment:
This needs to be attached to the floating component for the screen reader to announce it, so needs to be de-structured here